### PR TITLE
Adding Frosty Flake VIA keymap

### PR DESCRIPTION
**Pull request by trashcatt**
## Description

Added a VIA Layout for the Frosty Flake controller for the CoolerMaster QuickFire Rapid.
Not sure if anyone still has this keyboard, but I had one laying around and I wanted to see if I could port it to VIA.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bpiphany/frosty_flake/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/bpiphany/frosty_flake/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/bpiphany/frosty_flake/keymaps/via/keymap.c (hunk 1 above)
Comment (trashcatt) [reply keyboards/bpiphany/frosty_flake/keymaps/via/keymap.c:16]: Oops, completely forgot, fixed now.

**Review by drashna — APPROVED** [review]

File: keyboards/bpiphany/frosty_flake/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+       _______,_______,_______,                _______,                                 _______,    _______ , _______,  _______,   _______, _______, _______
+       ),
+
+[2] = LAYOUT_tkl(\
+       _______, _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______, _______,  _______,             _______, _______, _______, \
+       _______, _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______, _______,  _______,   _______,  _______, _______, \
+       _______, _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______, _______, _______,   _______,  _______,  _______, \
+       _______,_______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,_______,          _______, \
+       _______,_______,_______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,_______, _______,          _______,            _______, \
```
Comment (fauxpark) [inline keyboards/bpiphany/frosty_flake/keymaps/via/keymap.c:41]:
```suggestion
[2] = LAYOUT_tkl(
       _______, _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______, _______,  _______,             _______, _______, _______,
       _______, _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______, _______,  _______,   _______,  _______, _______,
       _______, _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______, _______, _______,   _______,  _______,  _______,
       _______,_______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,_______,          _______,
       _______,_______,_______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,_______, _______,          _______,            _______,
```

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

Thanks for contributing!

**Pull request merged**